client/src/apollo/apollo-links.js

```diff
@@ -8,7 +8,9 @@ const cookieXsrfToken = () => Cookies.get("XSRF-TOKEN")
 const fetchXsrfToken = async () => {
   return fetch("/sanctum/csrf-cookie", {
     credentials: "same-origin",
-  }).then(() => {
+  }).then(async (response) => {
+    //Read response text (even though its empty) to prevent Chrome from thinking there's an error b/c no one read the (empty) response body.
```

Comment:
```suggestion
    //Read response text (even though its empty) to prevent the browser from thinking there's an error b/c no one read the (empty) response body.
```